examples/raspi/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "raspi-oled"
+version = "0.1.0"
+authors = ["chux0519 <chuxdesign@hotmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+embedded-graphics = "0.6.0"
+linux-embedded-hal = "0.3.0"
+machine-ip = "0.2.1"
+ssd1306 = "0.4.0"
```

Comment:
I think this should be a `path` directive to keep the example code here the same as everywhere else:

```suggestion
# NOTE: The `path` directive is used here to keep the example up to date. Use a normal version
# number in real application code.
ssd1306 = { path = "../../" }
```